Sources/SQLiteNIO/SQLiteConnection.swift

```diff
@@ -200,6 +200,34 @@ public final class SQLiteConnection: SQLiteDatabase {
         return promise.futureResult
     }
 
+	public func install(customFunction: SQLiteCustomFunction) -> EventLoopFuture<Void> {
+		logger.trace("Adding custom function \(customFunction.name)")
+		let promise = self.eventLoop.makePromise(of: Void.self)
+		self.threadPool.submit { state in
+			do {
+				try customFunction.install(in: self)
+				promise.succeed(())
+			} catch {
+				promise.fail(error)
+			}
+		}
+		return promise.futureResult
```

Comment:
Per [the `NIOThreadPool` inline docs](https://github.com/apple/swift-nio/blob/main/Sources/NIOPosix/NIOThreadPool.swift#L130), use `runIfActive(eventLoop:_:)` instead:
```suggestion
        return self.threadPool.runIfActive(eventLoop: self.eventLoop) {
            try customFunction.install(in: self)
        }
```

This needs to be fixed pretty much everywhere in this file - this PR is an excellent opportunity to do so 🙂.

Reply:
Fixed. No more uses of `threadPool.submit` in the file